Dockerfile

```diff
@@ -6,4 +6,5 @@ WORKDIR /opt
 RUN pip install -r requirements.txt
 RUN mkdir /root/.aws
 
-ENTRYPOINT ["python", "scout.py"]
\ No newline at end of file
+ENTRYPOINT ["python", "scout.py"]
+CMD ["aws --no-browser --exceptions ifit-exceptions.js"]
```

Comment:
While this technically works, the [docs](https://docs.docker.com/engine/reference/builder/#cmd) follow pattern of either:

* `CMD ["executable","param1","param2"]` (exec form, this is the preferred form)
* `CMD command param1 param2` (shell form)

Reply:
`aws --no-browser --exceptions ifit-exceptions.js` are all params for `scout.py` but I was unable to get them to work when passed via ECS task container overide, can check localy again and confirm am all for following the docs if

```
ENTRYPOINT ["python", "scout.py"]
CMD ["aws","--no-browser","--exceptions ifit-exceptions.js"]
```

good call out @jonb-ifit it gets weirder too because I believe the correct Docker way **should** be:

```
ENTRYPOINT ["python"]
CMD ["scout.py","aws","--no-browser","--exceptions ifit-exceptions.js"]
```

🤔